templates/data/mongodb.html

```diff
@@ -283,8 +283,12 @@ <h3 class="p-heading--5">Consulting</h3>
               <li class="p-list__item is-ticked">Platform architecture and design</li>
               <li class="p-list__item is-ticked">Proof of Concept (PoC)</li>
               <li class="p-list__item is-ticked">Solution deployment</li>
-              <li class="p-list__item is-ticked">Charmed MongoDB training</li>
+              <li class="p-list__item is-ticked">
+                <a href="https://assets.ubuntu.com/v1/cfecb9cd-Charmed%20MongoDB%20Training_Public%20Module-4.pdf">Charmed MongoDB training</a>
```

Comment:
Could you confirm this link? The one on the doc is entitled Training_Public Module-3 whereas this one says Module-4. If the doc is out of date would you mind updating?

Reply:
yes, these were last minute changes from the stakeholder and also I had to put all the PDFs to Assets manager myself. I will update the copy doc.